sdk/developer-signing/Azure.Developer.Signing/ci.yml

```diff
@@ -0,0 +1,35 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
I overlooked where this is.  This should be in /sdk/developer-signing, not under the package.